src/Symfony/Component/Validator/Constraints/ImageValidator.php

```diff
@@ -43,6 +44,22 @@ public function validate($value, Constraint $constraint)
             return;
         }
 
+        if ($constraint->strict) {
+            if (!function_exists('imagecreatefromstring')) {
+                throw new RuntimeException('Strict image validation requires installed and enabled GD extension');
```

Comment:
Imo we should then change that in the `EmailValidator` too (if there isn't a reason for doing it this way that I am missing).